example/lib/widgets/buy_sheet.dart

```diff
@@ -68,7 +69,12 @@ class BuySheetState extends State<BuySheet> {
 
     switch (selection) {
       case PaymentType.cardPayment:
-        await _onStartCardEntryFlow();
+        if (_squareLocationSet) {
```

Comment:
Do I need location now to start a card entry flow?

Reply:
Yes, location id is required in order to call buyer verification flow.

Reply:
Can we find a better way to introduce SCA in the example. Feel override the existing cardEntry flow is a little bit arbitrary given most of developer doesn't care SCA at this moment.

Reply:
I am changing to call previous method as default. And add comments here so people interested in SCA flow can change the behavior if they want to test the SCA flow. In the future we could update example app to accommodate more scenarios but not for now.

Reply:
SG!